realm/realm-library/src/main/java/io/realm/internal/Table.java

```diff
@@ -52,10 +52,6 @@
     private final SharedRealm sharedRealm;
     private long cachedPrimaryKeyColumnIndex = NO_MATCH;
 
-    static {
```

Comment:
I'll create an new issue for figuring out what to do with tests. It is not entirely trivial, with also https://github.com/realm/realm-java/pull/3457#issuecomment-247944547